apps/client/src/features/rundown/event-editor/composite/EventEditorImage.tsx

```diff
@@ -0,0 +1,16 @@
+import style from './EventEditorImage.module.scss';
+
+interface EventEditorImageProps {
+  src: string;
+}
+
+export default function EventEditorImage(props: EventEditorImageProps) {
+  const { src } = props;
+
+  return (
+    <div className={style.imageContainer}>
+      <img loading='lazy' src={src} alt='' crossOrigin='anonymous' />
```

Comment:
```suggestion
      <img loading='lazy' src={src} alt={!src ? '' : ' '} crossOrigin='anonymous' />
```
if we did something like this the broken image icon would show if it fails to get the link

Reply:
good call